### DISCUSSION TITLE
Tailwind 3.0.8 and Grid

### DISCUSSION THREAD

**Discussion by Spacec4t** [tailwindlabs/tailwindcss#6838] [Help] [answered]
Hello, i have problems with using the Grid and Gap selectors in Tailwind 3.0.8. Is there a new selector for space between grid elements?
I.e.
`class="grid grid-cols-1 gap-12 sm:grid-cols-2 lg:grid-cols-3"`

doesnt work for me with the latest tailwind version

Thanks

**Comment by adamwathan** [comment, marked as answer]
Should work! Here's a working demo:

https://play.tailwindcss.com/wXgIngDrhN

Have you configured your `content` correctly?

https://tailwindcss.com/docs/content-configuration

**Reply by Spacec4t** [reply]
Thanks! Was just my fault not implementing the right line in the `content` section :-) Have a nice sunday
